compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/ClassReference.kt

```diff
@@ -84,6 +84,16 @@ public sealed class ClassReference {
    */
   public abstract fun enclosingClassesWithSelf(): List<ClassReference>
 
+  /**
+   * @param failedMessage The message included in the thrown Exception when this [ClassReference]
+   * doesn't have the expected visibility.
+   */
+  public fun checkHasVisibility(visibility: Visibility, failedMessage: String) {
```

Comment:
Initially I was going to include the error message here which made it feel worth having a dedicated method for this, but now this feels like not enough code to be worth it. Thoughts?

Reply:
I agree with your conclusion and I would probably inline this method again. You'll find several places in the codebase that use very similar error messages (especially when PSI or descriptors are mixed), but it was never reasonable to extract a common function. Sometimes having more flexibility and a little more duplication is the right thing. The gut feeling should decide :)

BTW, I was suggesting: 
```kotlin 
inline fun checkHasVisbility(visibility: Visibility: message: () -> String) { ... }
```